portfolio/src/main/webapp/script.js

```diff
@@ -63,11 +63,11 @@ function createMap() {
 
 function getComments() {
 	fetch('/data').then(response => response.json()).then((comments) => {
-		console.log(comments);
-		const bodyElement = document.body;
-		bodyElement.innerHTML += 'Here are the comments: \n';
-        for (comment of comments) {
-            bodyElement.innerHTML += comment + '\n';
-        }
+	  const bodyElement = document.body;
+          console.log('Comments: ', comments);
+          bodyElement.innerHTML += 'Here are all the comments: <br>';
+          for (comment of comments) {
+    		bodyElement.innerHTML += comment + '<br>';
+          }
 	});
```

Comment:
Mixing tabs and spaces is messy, you should either stick with the two-space indents or stick with tabs. Also in the other functions you had three-space indents.

Reply:
yup the tabs and spaces have been messed up from the beginning unfortunately. I use a mix of vim and editor so sometimes stuff gets wonky. I believe Chris is looking for a resource to assist with the spacing stuff :( Sorry about the inconvenience! Do you think we should block on this?